### PR TITLE
Project finished for correction

### PR DESCRIPTION
**Pull request by keikeka**

### CODE REVIEW

File: src/js/component/InputList.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		setList(newList);
+	};
+
+	//Function to delete task
+	const deleteTask = item => {
+		let deleteItem = [...list];
+		deleteItem.splice(item, 1);
+		setList(deleteItem);
+	};
+	//Returns the component using the Input property and adds the list of tasks using the Map method.
```
Comment (gaal1le0) [inline src/js/component/InputList.jsx]: Please remove this unnecessary comment :)

File: src/js/component/Items.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+//Component for added elements
+const Items = props => {
+	return (
+		<div className="p-3">
+			<li className="col-xl-12 bg-light">{props.name}</li>
+		</div>
+	);
+};
+
+//Proptypes validation
```
Comment (gaal1le0) [inline src/js/component/Items.jsx]: Please remove also this comment :)

File: src/js/component/Items.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+//Import REACT and PROPTYPES
+import React from "react";
+import PropTypes from "prop-types";
+
+//Component for added elements
```
Comment (gaal1le0) [inline src/js/component/Items.jsx]: Please remove also this comment :)

File: src/styles/InputList.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+.container {        
+    background: #fdfaf6;   
+    padding: 15%;
+    margin: auto;
+    font-family: 'EB Garamond', serif;    
+}
+// Style for input
+.form-control {
+    width: 60%;
+    margin-left: 10%;
```
Comment (gaal1le0) [inline src/styles/InputList.scss]: Remove all the comments, it's obvious that style is for a container and for a input :)

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,13 +1,8 @@
...
-    You can write your own styles here...
-    
-    
-    ...Or you can use the import command to include 
-    other sylesheets into your bunde as well, e.j:
-    
-    import 'relative/path/to/stylesheet.scss';
-
-*/
+// Specific style for ul
```
Comment (gaal1le0) [inline src/styles/index.scss]: Please remove too

File: src/js/component/InputList.jsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
+//Import REACT, HOOKS and PROPTYPES
```
Comment (gaal1le0) [inline src/js/component/InputList.jsx]: Do not include comments like these.

**Review by gaal1le0 — CHANGES_REQUESTED** [review]
STOP, please you must keep in mind that comments which not bring any IMPORTANT information to the reviewer mustn't be here. Please remove all the changes and request my review again to approve the MR.

File: src/js/component/InputList.jsx (hunk 1 above)
Comment (keikeka) [reply src/js/component/InputList.jsx]: Deleted all unnecessary comments. :)

**Review by gaal1le0 — APPROVED** [review]

**Pull request merged**